src/Microsoft.Azure.SignalR/HubHost/ServiceOptionsSetup.cs

```diff
@@ -8,11 +8,22 @@ namespace Microsoft.Azure.SignalR
 {
     internal class ServiceOptionsSetup : IConfigureOptions<ServiceOptions>
     {
+        private static readonly string ConnectionStringSecondaryKey =
```

Comment:
Did you test it on azure web apps?

Reply:
Yeah, tested. Works well.

Reply:
We should change this to exclude the trailing ConnectionString right?